web.test.tests/web.test.tests/Pages/HistoryPage.cs

```diff
@@ -0,0 +1,87 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace web.test.tests.Pages
+{
+    class HistoryPage
+    {
+        private IWebDriver driver;
+
+        public static String ClearBtnId = "clear";
```

Comment:
Looks like you don't need this variable. See next comment.

```suggestion
```

Reply:
Done